force-app/main/default/lwc/accountModelSettings/accountModelSettings.html

```diff
@@ -7,12 +7,81 @@
                 onsettingssaving={handleSettingsSaving}
                 onsettingssavecompleted={handleSettingsSaveCompleted}
             >
-                <!-- Body of Account Model Settings page (populates Slot in child component) -->
-                <lightning-button
-                    label="Test Button"
-                    disabled={affordancesDisabled}
-                ></lightning-button>
             </c-settings-save-canvas>
+
+            <!-- Body of Account Model Settings page (populates Slot in child component) -->
+            <template if:true={accountModelSettingsViewModel}>
+                <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
+                    <!-- Default Account Record Type Setting -->
+                    <div class="slds-col slds-size_2-of-3 slds-p-bottom_x-large">
+                        <div class="slds-text-title slds-text-color_default slds-p-left_small">
+                            {labelReference.stgAccModelTitle}
+                        </div>
+                        <div class="slds-text-body_small slds-text-color_weak slds-p-horizontal_large">
+                            {labelReference.stgHelpAccountModel}
+                        </div>
+                    </div>
+
+                    <div class="slds-col slds-size_1-of-3 slds-col_bump-right slds-p-right_x-large">
+                        <lightning-combobox
+                            name="defaultAccountModel"
```

Comment:
Keep these names in mind for navigation purposes. Also consider adding a data-qa-locator attribute by the same name so that QA can quickly navigate with robot. Again, this is a new item that came up from a discussion on Friday, so you could not have known about it beforehand.